onnxruntime/test/providers/vitisai/vitisai_basic_test.cc

```diff
@@ -0,0 +1,316 @@
+#pragma comment(lib, "d3d12.lib")
```

Comment:
That will not work on Linux. If the test is Windows only (I figured from the code), I suggest changing the file's name to state it explicitly. Otherwise, please move it to CMake instead for linking to that static library and make test os independent by moving Windows-specific code to a separate file and creating a standard API to Linux and Windows.